src/iden3comm/types/packer.ts

```diff
@@ -46,6 +49,16 @@ export type BasicMessage = {
   body?: unknown;
   from?: string;
   to?: string;
+  created_time?: number;
+  expires_time?: number;
+};
+
+/**
+ * Basic message with all possible fields required
+ */
+export type RequiredBasicMessage = Omit<Required<BasicMessage>, 'created_time' | 'expires_time'> & {
```

Comment:
Ohhhh. You do all fields  require  and then two optional... is okay of us? Why do we need this?